main.tf

```diff
@@ -13,21 +13,21 @@ resource "aws_lambda_function" "default" {
   # The function_name, runtime, memory_size, and timeout use variables
   # to facilitate testing of both new runtimes and new function versions.
   # End users will ordinarily use the default values.
-  function_name = "${var.function_name}"
+  function_name = var.function_name
 
-  runtime     = "${var.runtime}"
-  memory_size = "${var.memory_size}"
-  timeout     = "${var.timeout}"
+  runtime     = var.runtime
+  memory_size = var.memory_size
+  timeout     = var.timeout
   handler     = "index.handler"
-  publish     = "true"
-  role        = "${aws_iam_role.default.arn}"
+  publish     = true
+  role        = aws_iam_role.default.arn
   s3_bucket   = "drone-${local.region}-${local.account_id}"
```

Comment:
We need to do something about this hardcoded bucket.

Reply:
I filed an issue; see #3.